azure-functions/acbs-function/constants/industry.js

```diff
@@ -0,0 +1,9 @@
+const DEFAULT = {
+  ACBS: {
+    CODE: '1001',
+  },
+};
```

Comment:
What other properties would we put on these objects? I think it makes more sense to have
```suggestion
const DEFAULT_ACBS_INDUSTRY_CODE = '1001';
```